tests/Calculator.spec.ts

```diff
@@ -27,21 +27,21 @@ function mockAcceptedLaw(
     description: id,
     labels,
     treatAfterLabels,
-    effects,
+    effects: effects,
```

Comment:
Is that change needed?

Reply:
Not really. Don't know how it got in there.